src/components/Comments/Comments.js

```diff
@@ -0,0 +1,125 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import {
+  addCommentOfGoal,
+  fetchGoalComments,
+  archiveCommentOfGoal,
+  updateGoalComment,
+} from '../../goal-comments/actions'
+import moment from 'moment'
+import './Comments.css'
+import Avatar from '../Avatar/Avatar'
+import Button from '../Button/Button'
+function Comment(props) {
+  return (
+    <div className='comment'>
+      {console.log(props.avatarUrl)}
+      <div className='avatar_comment_container'>
+        <Avatar avatar_url={props.avatarUrl} />{' '}
+      </div>
+      <div>
+        <div className='info'>
+          <span>{props.name}</span>
+          <span className='date'>
+            {moment
+              .unix(props.unix)
+              .tz('America/Caracas')
+              .format('Do,hh:mm a')}
+          </span>
+        </div>
+        <span>{props.content}</span>
+      </div>
+    </div>
+  )
+}
+function Comments(props) {
```

Comment:
destructure the props here, for easy reading
```suggestion
function Comments({ goalAddress, avatarUrl, agents, comments, addCommentOfGoal }) {
```

and then inside the function call them by (for example)
`comments` instead of `props.comments`